packages/devtools/lib/src/tables.dart

```diff
@@ -628,13 +659,20 @@ abstract class Column<T> {
     ColumnAlignment alignment = ColumnAlignment.left,
     this.fixedWidthPx,
     this.percentWidth,
-  }) : _alignment = alignment {
+    this.wide = false,
+    this.usesHtml = false,
+    this.hover = false,
+    this.cssClass
+    }) : _alignment = alignment {
     if (percentWidth != null) {
       percentWidth.clamp(0, 100);
     }
   }
 
-  Column.wide(this.title, {ColumnAlignment alignment = ColumnAlignment.left})
+  Column.wide(this.title, {ColumnAlignment alignment = ColumnAlignment.left, this.wide = false,
```

Comment:
Why are we setting `wide` to false in the wide column constructor? why do we need the wide param in the first place?

If we do need it, it should be set to true in the body of this constructor and not be included in the params list.

Reply:
Not needed was there from before and when I was merging noticed you had removed - now its gone again.